### PR TITLE
Feature/rolling shutter led integration

### PR DESCRIPTION
**Pull request by psmithcrl**
This will allow the firmware to enable/disable the rolling shutter led synchronization, for the aux camera.

### CODE REVIEW

File: source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh [hunk 1]
```diff
@@ -2207,12 +2207,36 @@ public:
...
     }
+    /*
+    * Enable a rolling shutter camera flash synchronization, this will Allow
+    * an LED to flash with in sync with a rolling shutter imager, to reduce
+    * the possibility of, seeing inconsistent lighting artifacts with rolling
+    * shutter imagers.
+    *
+    * @param enabled enable/disable the rolling shutter synchronization feature.
+    *
+    * @return True on success, False on failure
```
Comment (mattalvarado) [inline source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh:2218]: I think we should add in details that this is only applicable for next generation cameras with a 3rd aux rolling-shutter color imager

File: source/LibMultiSense/include/MultiSense/details/wire/LedStatusMessage.hh [hunk 2]
```diff
@@ -122,7 +129,16 @@ public:
         }
         else
         {
-          invert_pulse = false;
+          invert_pulse = 0;
+        }
+
+        if (version >= 4)
+        {
+          message & invert_pulse;
```
Comment (mattalvarado) [inline source/LibMultiSense/include/MultiSense/details/wire/LedStatusMessage.hh:137]: should this be `rolling_shutter_led` rather than `invert_pulse`?

Comment (psmithcrl) [reply source/LibMultiSense/include/MultiSense/details/wire/LedStatusMessage.hh:137]: yes, it should.

File: source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh (hunk 1 above)
Comment (psmithcrl) [reply source/LibMultiSense/include/MultiSense/MultiSenseTypes.hh:2218]: Will do.

**Review by mattalvarado — APPROVED** [review]

**Pull request merged**